README.md

```diff
@@ -159,7 +159,7 @@ batch messaging is eliminated!
 If you are using a framework you might consider these composer packages to make the framework integration easier. 
 
 * [tehplague/swiftmailer-mailgun-bundle](https://github.com/tehplague/swiftmailer-mailgun-bundle) for Symfony2
-* [Bogardo/Mailgun](https://github.com/Bogardo/Mailgun) for Laravel 4
+* [Bogardo/Mailgun](https://github.com/Bogardo/Mailgun) for Laravel 5
```

Comment:
I also agree with @DavidGarciaCat, it would be best to just drop the Laravel version number here.